inference-engine/src/mkldnn_plugin/nodes/mkldnn_gather_node.cpp

```diff
@@ -2,24 +2,22 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-#include <cmath>
 #include <vector>
 #include <string>
 #include <mkldnn_types.h>
 #include "ie_parallel.hpp"
 #include "mkldnn_gather_node.h"
 #include <ngraph/opsets/opset1.hpp>
-#include <precision_utils.h>
 #include "common/cpu_memcpy.h"
 
 using namespace MKLDNNPlugin;
 using namespace InferenceEngine;
 
 bool MKLDNNGatherNode::isSupportedOperation(const std::shared_ptr<ngraph::Node>& op, std::string& errorMessage) noexcept {
     try {
-        auto gatherOp = ngraph::as_type_ptr<const ngraph::op::v1::Gather>(op);
+        auto gatherOp = ngraph::as_type_ptr<const ngraph::op::v7::Gather>(op);
         if (!gatherOp) {
-            errorMessage = "Only opset1 Gather operation is supported";
+            errorMessage = "Only opset7 Gather operation is supported";
             return false;
         }
 
```

Comment:
Looks like excessive validation of the GATHER_AXIS. NGraph also check that and we just use get_axis() below.

Reply:
Removed useless check. Thanks!